src/pages/BlockDetails.vue

```diff
@@ -0,0 +1,339 @@
+            />
+          </div>
+          <!-- This field not present -->
+          <div class="blocks-list-page__row">
+            <span class="h-sm cell">{{ $t('Transaction’s Merkle Root Hash') }}</span>
+          </div>
+          <div class="blocks-list-page__row">
+            <span class="h-sm cell">{{ $t('Rejected Transaction’s Merkle Root Hash') }}</span>
+            <BaseHash
+              :hash="block.rejected_transactions_merkle_root_hash"
+              :link="`/blocks/${block.height}`"
+              type="full"
+              copy
+              class="block-details-card__row__hash"
+            />
+          </div>
+          <div class="blocks-list-page__row">
+            <BaseLink :to="`/blocks/${block.height}`" class="block-details-card__row__hashFrame">
+              {{ $t('6 View Change Proofs') }}
+            </BaseLink>
+          </div>
+          <div class="blocks-list-page__row">
+            <BaseLink :to="`/blocks/${block.height}`" class="block-details-card__row__hashFrame">
+              {{ $t('13 Invalidated Block Hashes') }}
+            </BaseLink>
+          </div>
+        </div>
+      </template>
+    </BaseContentBlock>
+  </div>
+
+  <!-- dividing compoenent here  -->
+  <BaseContentBlock
+    :title="$t('Block transactions')"
+    class="transactions-list-page"
+  >
+    <div class="content-row">
+      <TransactionTypeFilter
+        v-model="tab"
+        class="transactions-list-page__tabs"
+      />
+      <TransactionStatusFilter
+        v-model="status"
+        class="transactions-list-page__status"
+      />
+    </div>
+
+    <BaseTable
+      :loading="table.loading.value"
+      :pagination="table.pagination"
+      :items="table.items.value"
+      container-class="transactions-list-page__container"
+      @next-page="table.nextPage()"
+      @prev-page="table.prevPage()"
+      @set-page="table.setPage($event)"
+      @set-size="table.setSize($event)"
+    >
+      <template #row="{ item }: { item: Transaction }">
+        <div class="transactions-list-page__row">
+          <TransactionStatus
+            type="tooltip"
+            class="transactions-list-page__icon"
+            :committed="item.committed"
+          />
+
+          <div class="transactions-list-page__column">
+            <div class="transactions-list-page__label">TxID</div>
+
+            <BaseHash
+              :hash="item.hash"
+              :type="hashType"
+              :link="'/transactions/' + item.hash"
+              copy
+            />
+
+            <div class="transactions-list-page__time">
+              {{ format(item.payload.creation_time) }}
+            </div>
+          </div>
+
+          <div class="transactions-list-page__column">
+            <div class="transactions-list-page__label">Block</div>
+
+            <BaseLink :to="'/blocks/' + item.block_height">
+              {{ item.block_height }}
+            </BaseLink>
+          </div>
```

Comment:
This column is probably redundant. All transactions displayed in this table already belong to the block which displays the table.

Reply:
This column is subject to discussion as per design for the ```BlockDetails Page```.